src/actions/ui.js

```diff
@@ -86,8 +86,8 @@ export function uiNotificationDelete(message) {
   return { type: UI_NOTIFICATION_DELETE, message }
 }
 
-export function uiInspectGetCuration(token, entity) {
-  return getCurationAction(token, entity, UI_INSPECT_GET_CURATION)
+export function uiInspectGetCurations(token, entity) {
+  return getCurationAction(token, entity, UI_INSPECT_GET_CURATIONS)
```

Comment:
If you are passing `UI_INSPECT_GET_CURATIONS` has a third param , i suggest to change `export function getCurationAction(token, entity)` to `export function getCurationAction(token, entity, name)`at https://github.com/clearlydefined/website/pull/515/files#diff-5aa8d7146713f00ce77323a034bcc4f9R14
instead of import the action name.

Reply:
makes sense 👍